bitsail-connectors/connector-assert/src/main/java/com/bytedance/bitsail/connector/assertion/sink/rule/AssertRuleParser.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.assertion.sink.rule;
+
+import com.bytedance.bitsail.common.BitSailException;
+import com.bytedance.bitsail.common.model.ColumnInfo;
+import com.bytedance.bitsail.common.typeinfo.TypeInfo;
+import com.bytedance.bitsail.common.typeinfo.TypeInfoBridge;
+import com.bytedance.bitsail.connector.assertion.sink.constants.AssertRuleType;
+import com.bytedance.bitsail.connector.assertion.sink.error.AssertErrorCode;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AssertRuleParser implements Serializable {
```

Comment:
IMO UT is necessary for this class. can we add it please

Reply:
sure, by the way, what's the meaning of IMO? :)

Reply:
it stands for in my opinion :)

Reply:
get it, thx